src/input/input_handler.cpp

```diff
@@ -0,0 +1,694 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "input_handler.h"
+
+#include "fstream"
+#include "iostream"
+#include "list"
+#include "map"
+#include "sstream"
+#include "string"
+#include "unordered_map"
+#include "vector"
```

Comment:
```suggestion
#include <vector>
```
Ditto for all other STL includes